### ISSUE TITLE
Integration test: kubelet fails to start

### ISSUE THREAD

**Issue by dkoshkin** [apprenda/kismatic#56]
```
13/16..  Install Kubernetes Kubelet on Worker Nodes                             
- Running task: kubelet : verify kubelet is running
  ip-10-0-3-174                                                                 [ERROR]
---- STDOUT ----
● kubelet.service - Kubernetes Kubelet
   Loaded: loaded (/etc/systemd/system/kubelet.service; enabled; vendor preset: enabled)
   Active: inactive (dead) since Mon 2016-11-07 05:44:26 UTC; 758ms ago
     Docs: https://github.com/GoogleCloudPlatform/kubernetes
  Process: 10988 ExecStart=/usr/bin/kubelet --allow-privileged=true --api-servers=${MASTER_LIST} --cloud-provider= --cluster-dns=${CLUSTER_DNS} --cluster-domain=cluster.local --configure-cbr0=true --container-runtime=docker --docker=unix:///var/run/docker.sock --hostname-override=${HOSTNAME} --kubeconfig=${KUBECONFIG} --pod-infra-container-image=${INFRA_IMG} --register-schedulable=${REGISTER_SCHEDULABLE} --serialize-image-pulls=false --tls-cert-file=${CERT_FILE} --tls-private-key-file=${KEY_FILE} --network-plugin=cni --network-plugin-dir=${NETWORK_PLUGIN_DIR} --v=2 (code=killed, signal=TERM)
 Main PID: 10988 (code=killed, signal=TERM)

Nov 07 05:44:26 ip-10-0-3-174 kubelet[10988]: I1107 05:44:26.220342   10988 kubelet_node_status.go:356] Recording NodeHasSufficientDisk event message for node ip-10-0-3-174
Nov 07 05:44:26 ip-10-0-3-174 kubelet[10988]: I1107 05:44:26.220361   10988 kubelet_node_status.go:356] Recording NodeHasSufficientMemory event message for node ip-10-0-3-174
Nov 07 05:44:26 ip-10-0-3-174 kubelet[10988]: I1107 05:44:26.220393   10988 kubelet_node_status.go:356] Recording NodeHasNoDiskPressure event message for node ip-10-0-3-174
Nov 07 05:44:26 ip-10-0-3-174 kubelet[10988]: I1107 05:44:26.220405   10988 kubelet_node_status.go:73] Attempting to register node ip-10-0-3-174
Nov 07 05:44:26 ip-10-0-3-174 kubelet[10988]: I1107 05:44:26.291343   10988 kubelet_node_status.go:76] Successfully registered node ip-10-0-3-174
Nov 07 05:44:26 ip-10-0-3-174 kubelet[10988]: I1107 05:44:26.322124   10988 kubelet_network.go:306] Setting Pod CIDR:  -> 172.16.0.0/24
Nov 07 05:44:26 ip-10-0-3-174 kubelet[10988]: I1107 05:44:26.484136   10988 container_bridge.go:81] CBR0 doesn't exist, attempting to create it with range: 172.16.0.1/24
Nov 07 05:44:26 ip-10-0-3-174 kubelet[10988]: I1107 05:44:26.515900   10988 manager.go:293] Recovery completed
Nov 07 05:44:26 ip-10-0-3-174 systemd[1]: Stopping Kubernetes Kubelet...
Nov 07 05:44:26 ip-10-0-3-174 systemd[1]: Stopped Kubernetes Kubelet.
---------------
error installing: error running playbook: error running ansible: exit status 1

• Failure [514.386 seconds]
Happy Path Installation Tests
/var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:194
  Installing against a minikube layout
  /var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:125
    Targeting AWS infrastructure
    /var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:113
      Using Ubuntu 16.04 LTS
      /var/snap-ci/src/github.com/apprenda/kismatic/integration/install_test.go:112
        should result in a working cluster [It]
        /var/snap-ci/src/github.com/apprenda/kismatic/integration/framework.go:16
```

**Comment by dkoshkin** [comment]
https://github.com/kubernetes/kubernetes/issues/12852 seems to be related.

I also wonder how https://github.com/kubernetes/kubernetes/pull/34906 will impact this, I've tested removing the `--configure-cbr0` flag from kubelet with `1.5.0-alpha` and all still works

**Comment by alexbrand** [comment]
It's interesting that there are no error messages about why the kubelet actually shuts down.

**Comment by alexbrand** [comment]
Closing as this was addressed in #65.
